plugins/auth/login.js

```diff
@@ -82,8 +88,15 @@ function addOAuthRoutes(config) {
                 const { collectionFactory } = request.server.app;
                 const accessToken = request.auth.credentials.token;
                 const { username } = request.auth.credentials.profile;
-
-                const profile = request.server.plugins.auth.generateProfile(username, scmContext, ['user'], {});
+                const scmUserId = request.auth.credentials.profile.id;
```

Comment:
where is `id` set?

Reply:
It is set by https://github.com/hapijs/bell plugin which is being used as the auth strategy for this route

Reply:
ah I see
https://github.com/hapijs/bell/blob/master/lib/providers/github.js#L26